### ISSUE TITLE
default learning_rate_frozen is too big

### ISSUE THREAD

**Issue by yangmingqi** [xcmyz/FastSpeech#37]
Thanks for author's work.
But I find the default learning_rate_frozen(1e-3) is too big, cause the loss can't convergence.
What is the learning_rate when you train the model?

**Comment by xcmyz** [comment]
train.py has been updated now.
